src/index.html

```diff
@@ -8,6 +8,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star stars__star--NA"></div>
+      <div class="stars__star stars__star--NA"></div>
+      <div class="stars__star stars__star--NA"></div>
+      <div class="stars__star stars__star--NA"></div>
+      <div class="stars__star stars__star--NA"></div>
```

Comment:
```suggestion
      <div class="stars__star stars__star"></div>
      <div class="stars__star stars__star"></div>
      <div class="stars__star stars__star"></div>
      <div class="stars__star stars__star"></div>
      <div class="stars__star stars__star"></div>
```
no modificators for star, only for `stars` block

Reply:
use nth-child to select propper background-image